test/test_python_errors.py

```diff
@@ -391,3 +391,26 @@ def test_repeated_kwarg():
 )
 def test_unparenthesized_genexp(source, no_errors):
     assert bool(_get_error_list(source)) ^ no_errors
+
+@pytest.mark.parametrize(
+    ('source', 'no_errors'), [
```

Comment:
FYI: I usually try to not double invert things, because it's hard to reason about. For example here `no_errors=False` would mean that there are errors, whereas `has_errors=True` and `has_errors=False` is easier to understand. But this is not an API, so it just doesn't matter at all. Just a thing to think about (which you probably already do, but I thought I'd post it anyways).